docs/testing.md

```diff
@@ -8,15 +8,15 @@
 
 ### Setup
 
-Testing Mithril applications is relatively easy. The easiest way to get started is with [ospec](https://mochajs.org/), [mithril-query](https://github.com/MithrilJS/mithril-query), and JSDOM. Installing those is pretty easy: open up a terminal and run this command.
```

Comment:
it's actually even simpler now, no need to add `jsdom`, since mithril-query uses `domino` internally and now comes with a auto-mocker for `window` and `raf`

Reply:
I propose to simply refer to the docs of `mithril-query` here, since changes can be made easier there.

Reply:
I'm game with that in a PR. I have opinions, but I'll hold them off until then.